Signal/test/call/PeerConnectionClientTest.swift

```diff
@@ -12,35 +12,37 @@ import WebRTC
 class FakePeerConnectionClientDelegate: PeerConnectionClientDelegate {
 
     enum ConnectionState {
-        case connected, failed
+        case connected, disconnected, failed
     }
 
     var connectionState: ConnectionState?
     var localIceCandidates = [RTCIceCandidate]()
     var dataChannelMessages = [OWSWebRTCProtosData]()
 
-    internal func peerConnectionClientIceConnected(_ peerconnectionClient: PeerConnectionClient) {
+    func peerConnectionClientIceConnected(_ peerconnectionClient: PeerConnectionClient) {
         connectionState = .connected
     }
 
-    internal func peerConnectionClientIceFailed(_ peerconnectionClient: PeerConnectionClient) {
+    func peerConnectionClientIceDisconnected(_ peerconnectionClient: PeerConnectionClient) {
+        connectionState = .disconnected
```

Comment:
Thanks for fixing up these tests.

Reply:
I just pushed one more commit to get these tests not just building but passing.

https://github.com/signalapp/Signal-iOS/pull/3482/commits/f7abcc9064eecc911f3fe9d3f10606dd8dcadc35